### ISSUE TITLE
Support chuvash characters

### ISSUE THREAD

**Issue by vovsemenv** [evilmartians/mono#23]
Hello, martians ! We would love to have chuvash language support in this font.
For chuvash language we only need to add this 4 characters in addition to russian. 
Additional characters list: 
- Ӑ ӑ
- Ӗ ӗ
- Ӳ ӳ
- Ҫ ҫ

cc @romashamin  from thread https://twitter.com/vovsy_/status/1612325861323153409

**Comment by vovsemenv** [comment]
Find out that maybe it's already supported current subset. I will check that again.

**Comment by vovsemenv** [comment]
I see similar glyphs in readme but in google fonts they dont work

<img width="1476" alt="image" src="https://user-images.githubusercontent.com/20096510/211342219-f1e5cd4a-7bc3-4dcb-8259-0cff48909801.png">

**Comment by romashamin** [comment]
Releasing the Cyrillic on GF is in progress: https://github.com/evilmartians/mono/pull/22

Thank you for the issue!

**Comment by ai** [comment]
Can we also re-use [Cyrillic Ҫ](https://ru.wikipedia.org/wiki/%D2%AA_(%D0%BA%D0%B8%D1%80%D0%B8%D0%BB%D0%BB%D0%B8%D1%86%D0%B0)) in [Latin Ç](https://ru.wikipedia.org/wiki/%C3%87_(%D0%BB%D0%B0%D1%82%D0%B8%D0%BD%D0%B8%D1%86%D0%B0)), which is used in Catalan? In some fonts they look very similar

**Comment by vovsemenv** [comment]
@ai yeah it's totally can be reused. At the first glance [Latin Ç](https://ru.wikipedia.org/wiki/%C3%87_(%D0%BB%D0%B0%D1%82%D0%B8%D0%BD%D0%B8%D1%86%D0%B0)) looks like perfect fit. Cyrillic one looks little bit weird for me because in all books bottom part was more round shapede (i mostly remember it from school program literature)

**Comment by emmamarichal** [comment]
Hi @vovsemenv,
I work in the GF team, I'm preparing Martian Mono for an upgrade, your request comes at the right time!
I added the glyphs needed for chuvash language support, you can find the good exports on my fork: 
https://github.com/emmamarichal/mono

Tell me if it's looks ok, so @romashamin can merge my PR.

**Comment by romashamin** [comment]
@vovsemenv it might be not easy to open Glyphs file so I made a little preview for you:
<img width="877" alt="image" src="https://user-images.githubusercontent.com/1148914/212663642-44dfd4cf-9321-4d6b-871b-cd9e0d44dea8.png">

What do you think?

**Comment by vovsemenv** [comment]
@romashamin  OMG this looks perfect !!!! Thank you for the good work 😻

**Comment by romashamin** [comment]
Great then! @vovsemenv thank you for your help 🙌

@emmamarichal, I believe we’re good to go!

**Comment by romashamin** [comment]
<img width="718" alt="image" src="https://user-images.githubusercontent.com/1148914/234943778-deef6fa0-3fb0-4c0f-8fa2-361cbd55f21d.png">

Note to self, the _Es with a descender_ should have their regular form with the straight tail. Also, the font should contain localized character variants with a cedilla in `Esdescender-cy.loclCHU` and `esdescender-cy.loclCHU` glyphs along with the next OT feature:

```
script cyrl;
language CHU;
lookup locl_cyrl_1 {
	sub Esdescender-cy by Esdescender-cy.loclCHU;
	sub esdescender-cy by esdescender-cy.loclCHU;
} locl_cyrl_1;
```

Will be fixed in 1.1.

**Comment by romashamin** [comment]
In Helvetica, the Chuvash glyph has a different shape from the regular cedilla.:
![GJBxlXjXIAAEO3q](https://github.com/evilmartians/mono/assets/1148914/e36f8483-07fc-4b5b-9c65-59c8c6b89ea9)

[Source](https://x.com/HuraVarman/status/1770039544412311782?s=20).
